ckan/values.yaml

```diff
@@ -1,189 +1,144 @@
-# Default values for ckan.
-# This is a YAML-formatted file.
-# Declare variables to be passed into your templates.
 general:
+  useProductionCerts: true
+  projectId: may-awesome-project
+  dockerconfigjson: SealedCredentialsForDockerCOntainerRegistry
+  imagePullSecrets: ckan-sealed-registry
+  customResource: true
+  resources: {}
   maintenance: false
-imagePullSecrets:
-  - name: ckan-sealed-registry
 
-nameOverride: ""
-fullnameOverride: ""
-
-serviceAccount:
-  # Specifies whether a service account should be created
-  create: true
-  # Annotations to add to the service account
-  annotations: {}
-  # The name of the service account to use.
-  # If not set and create is true, a name is generated using the fullname template
-  name: ""
-
-domain: ckan.staging.datopian.com
-useProductionCerts: false
+ingress:
+  enable: true
+  enableExternal: false
+  enableExternalCertificate: false
+  hosts:
+    - host: ckan.my-awesome-project.staging.datopian.com
+      isExternal: false
+      paths:
+        - path: /(.*)
+          service: may-awesome-project-ckan-svc
+          port: 80
+    - host: my-awesome-project.staging.datopian.com
+      isExternal: false
+      paths:
+        - path: /(.*)
+          service: may-awesome-project-frontend-svc
+          port: web
+  tls:
+    - secretName: ckan.my-awesome-project.staging.datopian.com
+      hosts:
+        - ckan.my-awesome-project.staging.datopian.com
+    - secretName: my-awesome-project.staging.datopian.com
+      hosts:
+        - my-awesome-project.staging.datopian.com
 
-datapusher:
-  enable: false
-  hpa:
-    enable: false
+solr:
   image:
-    repository: registry.gitlab.com/viderum/docker-datapusher
-    tag: cloud-datapusher-1-v9
-    pullPolicy: Always
-  resources:
-    limits:
-      cpu: 50m
-      memory: 300M
-    requests:
-      cpu: 50m
-      memory: 256M
+    tag: 6.6.6
 
 ckan:
-  useIni:
-    enable: false
-    iniPath: "/srv/app/production.ini"
-  hpa:
-    enable: false
-  resources:
-    limits:
-      cpu: 250m
-      memory: 1.2G
-    requests:
-      cpu: 200m
-      memory: 1G
```

Comment:
@shubham-mahajan also are we removing the default resource allocation of all the apps?